core/client/fs/src/main/java/alluxio/client/file/dora/DoraCacheClient.java

```diff
@@ -75,13 +74,13 @@ public DoraCacheFileInStream getInStream(URIStatus status,
                                            Protocol.OpenUfsBlockOptions ufsOptions) {
     WorkerNetAddress workerNetAddress = getWorkerNetAddress(status.getPath());
     // Construct the partial read request
-    DataReader.Factory readerFactory;
+    DoraDataReader reader;
     if (mNettyTransEnabled) {
-      readerFactory = createNettyDataReader(workerNetAddress, ufsOptions);
+      reader = createNettyDataReader(workerNetAddress, ufsOptions);
     } else {
-      readerFactory = createGrpcDataReader(workerNetAddress, ufsOptions);
+      throw new UnsupportedOperationException("Grpc dora reader not implemented");
```

Comment:
we will have a PR to deprecate GRPC in all data transport and at least in dora data transport?

Reply:
yes I think for data transport we are going to deprecate grpc